Updated date time calculation logic

oleksandrsarapulovgl opened the pull request.

fneur commented:
@oleksandrsarapulovgl
Strings like "2021-05-25T09:02:07" (with no time zone given at the end) don't parse successfully in `String.toFormattedDateTime` (throwing an exception during `ZonedDateTime.parse`) despite being ISO 8601 compliant. Do you see a problem in that?

(Imho the [specification](https://ec.europa.eu/health/sites/default/files/ehealth/docs/digital-green-certificates_dt-specifications_en.pdf) only mentions ISO 8601 compliant dates and times on page 5, but does not make time zone information mandatory)

oleksandrsarapulovgl commented:
You're right @fneur. Missed that, thank you for catching it! Updated logic to support date time with no time zone. Also, added unit tests to cover both cases: https://github.com/eu-digital-green-certificates/dgca-verifier-app-android/pull/56

fneur commented:
Thanks @oleksandrsarapulovgl
